testing/simapp/app.go

```diff
@@ -153,6 +154,7 @@ var (
 		crisis.AppModuleBasic{},
 		slashing.AppModuleBasic{},
 		ibc.AppModuleBasic{},
+		ibctm.AppModuleBasic{},
```

Comment:
is it worth adding a comment here to explicitly state that we are providing a module for the tendermint light client so the interfaces are registered? Maybe overkill, will leave it to you!

Reply:
Going to leave it for now. I think we can revisit in https://github.com/cosmos/ibc-go/issues/2008